R/pkg/R/sparkR.R

```diff
@@ -31,20 +31,27 @@ connExists <- function(env) {
 #' Stop the Spark context.
 #'
 #' Also terminates the backend this R session is connected to
+#' @export
 sparkR.stop <- function() {
+  .Deprecated("sparkR.session.stop")
+  sparkR.session.stop()
```

Comment:
Hmm - given that `sparkR.session.stop` and `sparkR.stop` have the same behavior, we can just modify the behavior `sparkR.stop` transparently ?